lib/slack.rb

```diff
@@ -1,11 +1,96 @@
 #!/usr/bin/env ruby
+# require_relative "../spec/test_helper"
+require_relative "channel"
+require_relative "user"
+require_relative "recipient"
+require_relative "workspace"
+require "dotenv"
+require "httparty"
+Dotenv.load
 
 def main
   puts "Welcome to the Ada Slack CLI!"
 
-  # TODO project
+  print "Channels loaded: ", Slack::Workspace.channel_list_all.length
+
+  print "\nUsers loaded: ", Slack::Workspace.user_list_all.length
+
+  puts "\nPlease select from the following options: \n list channels, select channel, list users, select user, details, send message, or quit"
+  selection = gets.chomp.downcase
+
+  until selection == "quit"
+    case selection
+    when "list channels"
+      puts Slack::Workspace.all_channels_details
+    when "select channel"
```

Comment:
The way you have this here means the channel doesn't **stay** selected.  It also means you have a lot of duplicate code in selecting users and channels and sending messages etc.